src/depth.rs

```diff
@@ -0,0 +1,74 @@
+use std::env;
+
+/// Supported color depths
+#[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Copy, Clone)]
+pub enum Depth {
+    /// 3-bit color
+    TTY = 3,
```

Comment:
Actually, isn't `Tty` preferred over `TTY` in Rust?

Reply:
it is, good point